csharp/ql/test/query-tests/Nullness/E.cs

```diff
@@ -411,6 +411,26 @@ int Ex41()
         i ??= null;
         return i.Value; // GOOD
     }
+
+    static bool Ex42(int? i, IEnumerable<int> @is)
+    {
+        return @is.Any(j => j == i.Value); // BAD (maybe)
+    }
+
+    static bool Ex43(int? i, IEnumerable<int> @is)
+    {
+        if (i.HasValue)
+            return @is.Any(j => j == i.Value); // GOOD
```

Comment:
Nice to get this covered!
I think that C#10 also has some improvements related to definite assignment in terms of compiler warnings. Not sure, if there is also some support for nullable values.